### PR TITLE
HPCC-15716 ECL Watch reports different information between new and legacy modes

### PR DESCRIPTION
**Pull request by kunalaswani**
WsWorkunits.js modified to report proper information on EventText

Signed-off by: Kunal Aswani kunal.aswani@lexisnexis.com

### CODE REVIEW

**Comment by kunalaswani** [comment]
@miguelvazq Please review

File: esp/src/eclwatch/WsWorkunits.js [hunk 1]
```diff
@@ -55,6 +66,7 @@ define([
             999: "not found"
         },
 
+
```
Comment (miguelvazq) [inline esp/src/eclwatch/WsWorkunits.js]: Minor comment: Remove extra line addition.

File: esp/src/eclwatch/WsWorkunits.js [hunk 2]
```diff
@@ -34,7 +34,18 @@ define([
...
+          action: "WUShowScheduled",
+          responseQualifier: "WUShowScheduledResponse.Workunits.ScheduledWU",
+          idProperty: "calculatedID",
 
+      preProcessRow: function (row) {
+          lang.mixin(row, {
+              calculatedID: row.Wuid + row.EventText
+          });
+      }
+  });
```
Comment (miguelvazq) [inline esp/src/eclwatch/WsWorkunits.js]: Indentation must line up with declaration opening.

**Comment by miguelvazq** [comment]
Looks good. FYI @GordonSmith

**Comment by miguelvazq** [comment]
@GordonSmith Please review as well.

File: esp/src/eclwatch/WsWorkunits.js [hunk 3]
```diff
@@ -32,9 +32,14 @@ define([
...
             action: "WUShowScheduled",
             responseQualifier: "WUShowScheduledResponse.Workunits.ScheduledWU",
-            idProperty: "Wuid"
-    });
+            idProperty: "calculatedID",
 
+        preProcessRow: function (row) {
+            lang.mixin(row, {
+                calculatedID: row.Wuid + row.EventText
+            });
```
Comment (GordonSmith) [inline esp/src/eclwatch/WsWorkunits.js]: Minor Comment:  ```row.calculatedID = row.Wuid + row.EventText;``` would have been equally good (but you may have needed to check if "row" existed).

**Comment by GordonSmith** [comment]
Looks fine.

**Pull request merged**